http/server.go

```diff
@@ -0,0 +1,80 @@
+package http
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/drand/drand/protobuf/drand"
+)
+
+// New creates an HTTP handler for the public Drand API
+func New(ctx context.Context, client drand.PublicClient) (http.Handler, error) {
+	pkt, err := client.Group(ctx, &drand.GroupRequest{})
```

Comment:
You could use the conversion from the proto packet to the real struct as here: https://github.com/drand/drand/blob/master/core/convert.go#L17
Then you would have the real `key.Group` [struct](https://github.com/drand/drand/blob/master/key/group.go#L26)

Reply:
i'll note some confusion between the division of code in core vs key vs net vs beacon :)